.github/workflows/ci.yml

```
@@ -230,3 +230,20 @@ jobs:
               cmake --build build/Debug --target ${file%.cpp}.o
             fi
           done
+
+  linterAll:
+    name: Build/Run linter on all files
+    needs: dockerImageUnitTests
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/genspectrum/lapis-silo-dependencies:commit-${{ github.sha }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - shell: bash
+        name: Configure and run clang-tidy on changed files
+        run: |
+          mv /src/build .
+          cmake -DBUILD_WITH_CLANG_TIDY=on -D CMAKE_BUILD_TYPE=Debug -B build/Debug
```

Does the space make a difference?
```suggestion
          cmake -D BUILD_WITH_CLANG_TIDY=on -D CMAKE_BUILD_TYPE=Debug -B build/Debug
```